packages/components/src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx

```diff
@@ -14,29 +14,34 @@ const ContentPageHeader = ({
   LinkComponent,
 }: ContentPageHeaderProps) => {
   return (
-    <div className="bg-site-primary-100">
-      <div className="mx-auto flex max-w-[880px] flex-col gap-8 px-6 py-8 md:px-10 lg:max-w-container lg:gap-12 lg:py-16">
-        <div className="hidden lg:block">
-          <Breadcrumb links={breadcrumb.links} LinkComponent={LinkComponent} />
+    <div className="bg-site-secondary-100">
+      <div className="max-w-container mx-auto">
+        <div className="flex max-w-[880px] flex-col gap-8 px-6 py-8 md:px-10 lg:gap-12 lg:py-16">
```

Comment:
nit: define a max x rather than inlining this

Reply:
i think we're doing this for all layouts atm! will leave it to one shot fix the layouts with a max width